extended-statistics/src/test/java/org/infinispan/stats/BaseClusteredExtendedStatisticTest.java

```diff
@@ -435,13 +436,13 @@ private void putIfAbsent(int cacheIndex, Object key, Object value) throws Interr
 
    private Object replace(int cacheIndex, Object key, Object value) throws InterruptedException {
       Object val = cache(cacheIndex).replace(key, value);
-      awaitReplace(cacheIndex, key);
+      awaitPut(cacheIndex, key);
```

Comment:
Some commands are not replicated to backups as-is; the replace command requires at least some previous value but it would be wasteful to fetch it (as we're going to overwrite the value anyway and events on backup owners don't have correct prev values guaranteed).
So in this case ReplaceCommand is invoked on primary and backup gets just PutKeyValueCommand.